fix: attacked 로직 수정

rlaguswls2 opened the pull request:
#11

rlaguswls2 committed:
fix: attacked 로직 수정

#11

ShrimpManiac reviewed (commented):
[2024/12/18 14:42] 수정 요망
- `isDead` 컨디션 오류 수정
- `sendPacket` opponent & user 뒤바뀜 수정

ShrimpManiac committed:
rename: 패킷명세 이름변경

- `unitAttacked` -> `damageUnit`
- `baseAttacked` -> `damageBase`
- `attackedUnit` -> `targetUnit`

Ref: #11, #14, #75

ShrimpManiac committed:
refactor: 유닛 사망 패킷 통합

- 상대방 알림을 삭제하고 응답 패킷이 역할을 대신

Ref: #12, #75

ShrimpManiac committed:
refactor: 유닛 공격/피격 패킷 통합

- 상대방 공격/피격 알림 패킷을 삭제하고 그 역할을 공격/피격 알림을 대신

Ref: #11, #75

ShrimpManiac committed:
fix: 성채공격 응답에 `success` 필드를 보내주도록 변경

Ref: #14, #75

ShrimpManiac committed:
rename: `isAttackAvailable` -> `isAttackOnCooldown`

Ref: #11

ShrimpManiac committed:
fix: 유닛 공격 패킷에 `success` 필드 추가

Ref: #11, #75

ShrimpManiac committed:
Fix: 사거리 오차허용범위 변경 5% -> 50%

Ref: #11, #61

ShrimpManiac committed:
comment: 필요없는 주석 삭제

ShrimpManiac reviewed (approved):
[2024/12/18 17:27] 2인 리뷰 완료
